src/game_renderer/GameRenderer.cpp

```diff
@@ -83,7 +79,7 @@ void GameRenderer::Init(const std::string& resource_dir,
   std::cout << "OpenGL version: " << glGetString(GL_VERSION) << std::endl;
   std::cout << "GLSL version: " << glGetString(GL_SHADING_LANGUAGE_VERSION)
             << std::endl;
-  glfwSwapInterval(0);  // vsync
+  glfwSwapInterval(1);  // vsync
```

Comment:
Any time you can run more than 60fps or whatever your monitor refresh rate is, you would want it on so that you arent needlessly overworking your graphics card. Without it I get like 1700fps, but I still only see 60 of the 1700